collector/__init__.py

```diff
@@ -0,0 +1,590 @@
+from zipfile import ZipFile
+import xml.etree.cElementTree as ElementTree
+
+from astropy.coordinates import SkyCoord, EarthLocation
+import astropy.units as u
+from astropy.table import Table, vstack
+
+import pytz
+import os      
+import calendar
+import numpy as np
+
+import collector.sb as sb
+from collector.vskyutil import nightevents
+from collector.xmlutils import *
+from collector.get_tadata import get_report, get_tas, sumtas_date
+from collector.conditions import SkyConditions, WindConditions
+
+from greedy_max.instrument import Instrument
+from greedy_max.site import Site
+from greedy_max.band import Band
+from greedy_max.schedule import Observation
+from greedy_max.category import Category
+
+from typing import List, Dict, Optional, NoReturn
+
+MAX_AIRMASS = '2.3'
+
+   
+def uniquelist(seq):
+    # Make a list of unique values
+    # http://stackoverflow.com/questions/480214/how-do-you-remove-duplicates-from-a-list-in-python-whilst-preserving-order
+    seen = set()
+    seen_add = seen.add
+    return [x for x in seq if not (x in seen or seen_add(x))]
```

Comment:
I meant to delete that method. I think is still being used in some XML parsing functions returns so that's why it wasn't. Like you said Python can do it with dicts, I use that implementation instead further in the code.